codec_specs.md

```diff
@@ -463,8 +463,10 @@ Description: VP9 is an open and royalty free video compression format developed
 
 Codec BlockAdditions: A single-channel encoding of an alpha channel **MAY** be stored in `BlockAdditions`. The `BlockAddId` of the `BlockMore` containing these data **MUST** be 1.
 
-Initialization: The `CodecPrivate` **MUST** contain a list of specific VP9 codec features as described in the "VP9 Codec Feature Metadata" section of [@!WebMContainer].
-Due to the existence of some old non compliant muxers which were not providing `CodecPrivate`, the decoder **SHOULD** reconstruct it from the first frame in the case `CodecPrivate` is not present.
+Initialization: The `CodecPrivate` **SHOULD** contain a list of specific VP9 codec features as described in the "VP9 Codec Feature Metadata" section of [@!WebMContainer].
+Many muxers don't provide the `CodecPrivate` for "V_VP9" so it's not a hard requirement. However it is **RECOMMENDED** to provide this information that can help select a decoder on playback.
+It is possible for the decoder to reconstruct the "VP9 Codec Feature Metadata" from the first frame in the case `CodecPrivate` is not present.
```

Comment:
Minor nitpicks:

- "…in the case…" is usually used simply as "…in case…"
- The whole sentence "Many muxers don't…" might read better as: "As many muxers don't provide the `CodecPrivate` for "V_VP9" tracks, it's not a hard requirement."
- I think there should be a comma after the "However" in "However it is…", as in: "However, it is…"

Apart from these I'm fine with the change. Thanks.

Reply:
I made the suggested changes. We do have a comma after all the "However" usage.